### PR TITLE
Hyd 3 access cmi from hydroid

### PR DESCRIPTION
**Pull request by deepikm**
Implementation for searching cmi nodes from hydroid. 
Todo
- parse author name from json, once exposed in cmi endpoint
- Clean up code that reads data from file (left in there from testing purposes)

### CODE REVIEW

File: src/main/java/au/gov/ga/hydroid/service/impl/EnhancerServiceImpl.java [hunk 1]
```diff
@@ -416,6 +424,65 @@ public void enhanceImages() {
       }
    }
 
+    @Override
+    public void enhanceCMINodes() {
```
Comment (scelton) [inline src/main/java/au/gov/ga/hydroid/service/impl/EnhancerServiceImpl.java]: If you can, please make this method a little smaller, maybe creating other methods... not a big issue if the logic doesn't allow for it.

**Comment by scelton** [comment]
Looks great! Just added a small comment about the enhanceCMINodes()

File: src/main/java/au/gov/ga/hydroid/service/impl/EnhancerServiceImpl.java (hunk 1 above)
Comment (deepikm) [inline src/main/java/au/gov/ga/hydroid/service/impl/EnhancerServiceImpl.java]: I guess EnhancerServiceImpl class as a whole has to be refactored using helper classes. But for now have refactored the enhanceCMINodes into smaller methods.

**Comment by deepikm** [comment]
new Commit  - incorporates the review comments from above and also includes implementation for parsing author information.

**Comment by scelton** [comment]
Looks good :thumbsup:

**Pull request merged**